src/chapters/06-agents.adoc

```diff
@@ -79,11 +81,11 @@ build once, fingerprint and then reuse
 * Challenge: workspace is *NOT* shared across agents.
 ** A new workspace is allocated for *each* stage
 
-* Pipeline can "pass" artifacts from stage to stage
+* Pipeline can move artifacts from stage to stage
 ** Assumption: building a binary is 1 order of magnitude slower
 than copying it on the network
 
-* 2 types of "passing artifacts", both are using the master:
+* 2 Artifact's types, both are using the master:
```

Comment:
This is wrong semantically. The artifacts are files generated by the builds.
Here this is more the way the Pipeline pass/move/reuse/copy the artifacts from a stage to another.

However you are right to fix this, the original sentence feels wrong. 

What do you think about this one?

```
Two ways of storing artifacts
```

Reply:
I don't understand how different it is from 'Pipeline can reuse artifacts'